setup_codecov.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -ex
```

Comment:
the `x` is risky, isn't it? Why not the usual 
```suggestion
set -euo pipefail
```

Reply:
Good point !